src/styles/blocks/card.scss

```diff
@@ -0,0 +1,69 @@
+
+.card {
+  margin-right: 48px;
+  margin-bottom: 48px;
+  box-sizing: border-box;
```

Comment:
we cant use margine in BEM
[BEM & STYLES] - Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.